package.json

```diff
@@ -11,6 +13,35 @@
     "start": "node ./index.js",
     "test": "run-s lint jest"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/renovatebot/renovate-approve-bot-bitbucket-cloud.git"
+  },
+  "keywords": [
+    "bitbucket",
+    "dependencies",
+    "dependency",
+    "docker",
+    "management",
+    "outdated",
+    "update"
+  ],
+  "author": "Maxime Brunet <insert-email-here>",
+  "contributors": [
+    "HonkingGoose",
+    "Michael Kriese <michael.kriese@visualon.de>"
+  ],
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/renovatebot/renovate-approve-bot-bitbucket-cloud/issues"
+  },
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "engines-next": {
+    "description": "Versions other than the below are deprecated and a warning will be logged",
+    "node": ">=14.15.0"
+  },
```

Comment:
Will need some help to set this up correctly. I've copy/pasted the `engines` and `engines-next` fields from the Renovate repository `package.json`.

Reply:
engines-next can be removed